packages/ingest/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@stackpress/types": "0.2.10",
-    "cookie": "1.0.2",
+    "cookie": "^1.0.2",
```

Comment:
Just to note: I dont use `^` because some authors (like myself) deploy breaking changes. Removing this guarantees stability for everyone that uses this lib.